### PR TITLE
New landing page for charts

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #183, #281 and #295.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/bar.md [hunk 1]
```diff
@@ -41,34 +57,30 @@ is displayed on the page (the blue color is not related to what party the
 elected president belongs to - it is just the default color that Plotly
 has picked up):
 
-![Bar chart](bar1.png)
-
-!!! note "Faint bars"
-    Plotly gives bars a low opacity unless they are selected. If you
-    want to override that behavior and force a higher opacity, you
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md:48]: good catch

File: docs/manuals/gui/viselements/charts/bar.md [hunk 2]
```diff
@@ -41,34 +57,30 @@ is displayed on the page (the blue color is not related to what party the
...
-    }
-    ```
+<figure>
+    <img src="../bar-simple-d.png" class="visible-dark" />
+    <img src="../bar-simple-l.png" class="visible-light"/>
+    <figcaption>Simple bar chart</figcaption>
+</figure>
 
-### Multiple data sets
+### Multiple data sets {data-source="gui:doc/examples/charts/bar-multiple"}
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md:66]: I like this data-source

File: docs/manuals/gui/viselements/charts/bar.md [hunk 3]
```diff
@@ -41,34 +57,30 @@ is displayed on the page (the blue color is not related to what party the
...
+data = pandas.DataFrame(percentages, columns= ["Year", "Won"])
+# Add the Lost column (100-Won)
 data["Lost"] = [100-t[1] for t in percentages]
 ```
 
 We add a new column to the DataFrame _data_, which is the complement to 100
 of _percentages_.
 
-To represent it, we will change the definition of the cart control:
+To represent it, we will change the definition of the cart control, splitting the two
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md]: cart => chart ?

File: docs/manuals/gui/viselements/charts/bar.md [hunk 4]
```diff
@@ -176,8 +201,8 @@ _orientation[]_ property is used to change the orientation of the bars in
 our chart.
 
 Also, note that the _layout_ property is set as well: We indicate in its
-_barmode_ property that the two charts should be displayed next to each
-other, and we hide the legend using the _showlegend_ property.
+_barmode_ property that the two charts should be share the same y coordinates,
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md]: should be share => should share ?

File: docs/manuals/gui/viselements/charts/scatter.md [hunk 5]
```diff
@@ -1,9 +1,24 @@
...
+[_mode[]_](../chart.md#p-mode) property for your trace to "markers".
+
+## Key properties
+
+| Parameter       | Value             | Notes |
+| --------------- | ------------------------- | ------------------ |
+| [_mode_](../chart.md#p-mode)      | `markers`          |  |
+| [_x_](../chart.md#p-x)            | x values           |  |
+| [_y_](../chart.md#p-y)            | y values           |  |
+| [_marker_](../chart.md#p-marker)  | dictionary  | `size` can be an integer or a column name in _data_.<br/>`color` can be a color value or a column name in _data_.<br/>`symbol` must be a predefined symbol name. |
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/scatter.md]: "color", "size", "symbol", "opacity" can be column names

**Review by FredLL-Avaiga — APPROVED** [review]
What's the point of text png ? 
- home-2d-hoistogram-d.png 
- ...

File: docs/manuals/gui/viselements/charts/bar.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bar.md]: No no. Cart.
Hum just kidding

**Comment by FabienLelaquais** [comment]
> 

These text png files are placeholders so that he doc looks ok for the time begin, waiting to chart type home images to be generated.

File: docs/manuals/gui/viselements/charts/scatter.md (hunk 5 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/scatter.md]: 👍

File: docs/manuals/gui/viselements/charts/bar.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bar.md]: 👍

File: docs/manuals/gui/viselements/charts/bar.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bar.md:66]: 👌

File: docs/manuals/gui/viselements/charts/bar.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bar.md:48]: 👌

**Pull request merged**